### PR TITLE
Add helm/nginx chapter, fix JS console exception

### PR DESCRIPTION
**Pull request by geremyCohen**
Addresses issues:
https://github.com/aws-samples/eks-workshop/issues/107
https://github.com/aws-samples/eks-workshop/issues/120

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/helm-nginx/addbitnamirepo.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+bitnami/nginx-ingress-controller        2.1.4           0.20.0          Chart for the nginx Ingress...
+```
+
+In both of those last two searches, we see
+
+```
+bitnami/nginx
+```
+
+as a search result.  That's the one we're looking for, so lets next use Helm to install it to the EKS cluster.
```
Comment (dalbhanj) [inline content/helm-nginx/addbitnamirepo.md]: so lets next use (so lets use)

File: content/helm-nginx/installnginx.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+Upon running this command, you should see some confirmations around the objects that were created:
+
+```
+NAME:   mywebserver
+LAST DEPLOYED: Tue Nov 13 19:55:25 2018
+NAMESPACE: default
+STATUS: DEPLOYED
+
+RESOURCES:
+==> v1beta1/Deployment
```
Comment (dalbhanj) [inline content/helm-nginx/installnginx.md]: Can you talk about resources that are created? I can see this helm chart creates deployment, service etc.

File: content/helm-nginx/installnginx.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+NAME               AGE
+mywebserver-nginx  0s
+
+==> v1/Pod(related)
+
+NAME                                READY  STATUS             RESTARTS  AGE
+mywebserver-nginx-85985c8466-tczst  0/1    ContainerCreating  0         0s
+
+==> v1/Service
```
Comment (dalbhanj) [inline content/helm-nginx/installnginx.md]: I would also point to helm recommendations for things to check while it creates resources. You can ignore this if it doesn't apply to nginx chart

File: content/helm-nginx/cleaningup.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+```
+NAME            REVISION        UPDATED                         STATUS          CHART           APP VERSION     NAMESPACE
+mywebserver     1               Tue Nov 13 19:55:25 2018        DEPLOYED        nginx-1.1.2     1.14.1          default
+```
+
+It was a lot of fun; we had some great times sending HTTP back and forth, but now its time to delete this deployment.  To delete:
+
+```
+helm delete mywebserver
```
Comment (dalbhanj) [inline content/helm-nginx/cleaningup.md]: don't you need to use '--purge' to delete helm chart completely?

**Comment by brentley** [comment]
I'm going to cherry-pick the js fix into a different PR and merge it.

File: content/helm-nginx/cleaningup.md (hunk 4 above)
Comment (geremyCohen) [reply content/helm-nginx/cleaningup.md]: fixed

File: content/helm-nginx/installnginx.md (hunk 3 above)
Comment (geremyCohen) [reply content/helm-nginx/installnginx.md]: there don't appear to be any

File: content/helm-nginx/_index.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+# Deploy NGINX with Helm
+
+As we saw previously in the [Helm Chapter](../helm), [Helm](https://helm.sh/) is a tool for managing Kubernetes charts. Charts are packages of pre-configured Kubernetes resources.
+
+In this Chapter, we will dig deeper with Helm and demonstrate how to install the NGINX web server by showing you how to:
+
+{{% children showhidden="false" %}}
+
+Be sure you completed the the chapter [Deploy Helm](../deploy) before continuing, as having Helm installed is a prerequisite for completing this chapter.
```
[the remark this answers is not in the document]
Comment (geremyCohen) [reply content/helm-nginx/_index.md]: fixed

File: content/helm-nginx/addbitnamirepo.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
+---
+title: "Add the Bitnami Repository"
+date: 2018-08-07T08:30:11-07:00
+weight: 30
+---
+
+In the last slide, we saw that NGINX offers many different products via the default Helm Chart repository, but the NGINX standalone web server is not one of them.
```
[the remark this answers is not in the document]
Comment (geremyCohen) [reply content/helm-nginx/addbitnamirepo.md]: yes, thats correct. its a segue from the last slide.

File: content/helm-nginx/_index.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+
+# Deploy NGINX with Helm
+
+As we saw previously in the [Helm Chapter](../helm), [Helm](https://helm.sh/) is a tool for managing Kubernetes charts. Charts are packages of pre-configured Kubernetes resources.
+
+In this Chapter, we will dig deeper with Helm and demonstrate how to install the NGINX web server by showing you how to:
+
+{{% children showhidden="false" %}}
+
+Be sure you completed the the chapter [Deploy Helm](../helm) before continuing, as having Helm installed is a prerequisite for completing this chapter.
```
[the remark this answers is not in the document]
Comment (geremyCohen) [reply content/helm-nginx/_index.md]: fixed

File: content/helm-nginx/addbitnamirepo.md (hunk 1 above)
Comment (geremyCohen) [reply content/helm-nginx/addbitnamirepo.md]: fixed

File: content/helm-nginx/installnginx.md (hunk 2 above)
Comment (geremyCohen) [reply content/helm-nginx/installnginx.md]: fixed

**Pull request merged**